### PR TITLE
sequence similarity visualization

### PR DESCRIPTION
**Pull request by chuntul**

### CODE REVIEW

File: R/seqSimVis.R [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                      mart = ensembl)$peptide
+  for (seq in seq1) {
+    if (seq != "Sequence unavailable") {
+      seq1 <- seq
+      break
+    }
+  }
+
+  # get genes of system
+  sys_genes <- fetchComponents(system)
```
Comment (rn1n3r) [inline R/seqSimVis.R:36]: fetchComponents() is deprecated, please use `SyDBgetSysSymbols()`.

File: R/seqSimVis.R [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+#'
+#'
+#' @param gene_of_int (char) gene of interest
+#' @param system (char) system of interest
+
+#' @examples
+#' seqSimVis("BRCA1", "PHALY")
+#'
+#' @export
+seqSimVis <- function(gene_of_int, system) {
```
Comment (rn1n3r) [inline R/seqSimVis.R:14]: Consider having the function return the similarity values for further analysis/visualization if necessary.

File: BCB420.2019.ESA.Rproj [hunk 3]
```diff
@@ -18,3 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace,vignette
```
Comment (rn1n3r) [inline BCB420.2019.ESA.Rproj:21]: I don't think you need to add this for roxygenize to work. See if removing it changes anything

File: R/seqSimVis.R [hunk 4]
```diff
@@ -0,0 +1,98 @@
+
+
+#' \code{seqSimVis()} Calculates sequence similarity and plots dot plots based on the
+#' three highest similarity values to visualize
```
Comment (rn1n3r) [inline R/seqSimVis.R:4]: Maybe you could specify the number of top similarity values to visualize as a parameter for your function, and set 3 to be the default value.

File: DESCRIPTION [hunk 5]
```diff
@@ -19,7 +24,13 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
-    Biostrings
+    Biostrings,
+    BiocManager,
+    biomaRt,
+    protr,
+    doParallel,
```
Comment (rn1n3r) [inline DESCRIPTION:43]: Not sure if you actually use doParallel or foreach in your code.

File: R/seqSimVis.R [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+#'
+#'
+#' @param gene_of_int (char) gene of interest
+#' @param system (char) system of interest
+
+#' @examples
+#' seqSimVis("BRCA1", "PHALY")
```
Comment (rn1n3r) [inline R/seqSimVis.R:11]: Please add a comment explaining what your example code does.

**Review by rn1n3r — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 7]
```diff
@@ -1,2 +1,3 @@
 # R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
```
Comment (hyginn) [inline .travis.yml:1]: This file should not have been changed and committed by you.

File: DESCRIPTION (hunk 5 above)
Comment (hyginn) [reply DESCRIPTION:43]: You "resolved" this conversation without a comment or response.

File: R/seqSimVis.R (hunk 4 above)
Comment (hyginn) [reply R/seqSimVis.R:4]: Again - I don't see how this was "resolved" ... :-(

File: R/seqSimVis.R (hunk 6 above)
Comment (hyginn) [reply R/seqSimVis.R:11]: ... so: where is the comment?

**Review by hyginn — COMMENTED** [review]
I don't really see the working hypothesis here. No examples are shown, no results interpreted. Vignette is missing. Reviewer comments were not addressed. Marked as-is.

**Comment by hyginn** [comment]
Won't merge, marked as-is. Closed.

**Pull request closed without merging**